src/kvazaar.h

```diff
@@ -34,6 +34,10 @@
 extern "C" {
 #endif
 
+#if defined(KVZ_STATIC_LIB) && defined(PIC)
+  #undef KVZ_STATIC_LIB
```

Comment:
Why is this needed? It causes linking against a static kvazaar lib to fail if the user has defined `PIC`.